src/components/general/modals/share-experiences-modal.js

```diff
@@ -59,6 +59,8 @@ export default function ShareExperiencesModal(props) {
     setOpen(isOpen);
     if (isOpen === true) {
       setScroll(scrollType);
+    } else if(currentScreen === 3){
```

Comment:
If the modal is closing and the current screen is the final screen, then the page will reload